test/ocsp/helper/helper.go

```diff
@@ -222,5 +258,14 @@ func parseAndPrint(respBytes []byte, cert, issuer *x509.Certificate, expectStatu
 	fmt.Printf("  RevocationReason %d\n", resp.RevocationReason)
 	fmt.Printf("  SignatureAlgorithm %s\n", resp.SignatureAlgorithm)
 	fmt.Printf("  Extensions %#v\n", resp.Extensions)
+	if resp.Certificate == nil {
+		fmt.Printf("  Certificate: nil\n")
+	} else {
+		fmt.Printf("  Certificate:\n")
```

Comment:
extremely nit: Printf with no formatting directive.

Reply:
somewhat surprised we don;t get a vet error for this.

Reply:
go vet actually does spot this, but we don't run go vet in Travis because it's too noisy, and it's explicitly documented as something not recommended for CI.

But thanks for the suggestion on running it- I actually found a bug on another line.